### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Globalika**
added 3rd party libraries with cocoapods
added swiftlint file
made read me instruction
set up git ignore 
integrate apollo client with api https://countries.trevorblades.com/

### CODE REVIEW

File: Countries/Countries/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            do {
+                try context.save()
+            } catch {
+                // Replace this implementation with code to handle the error appropriately.
+                // fatalError() causes the application to generate a crash log and terminate. You should not use this function in a shipping application, although it may be useful during development.
+                let nserror = error as NSError
+                fatalError("Unresolved error \(nserror), \(nserror.userInfo)")
+            }
+        }
+    }
```
Comment (apolwork) [inline Countries/Countries/AppDelegate.swift:78]: Do you plan to use CoreData in your application?

File: Countries/Countries/Queries.graphql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+query Test{
+  continents {
+    name
+  }
+}
```
Comment (apolwork) [inline Countries/Countries/Queries.graphql:5]: Do you have plans to replace this query with more appropriate?

File: Countries/Countries/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+
```
Comment (apolwork) [inline Countries/Countries/ViewController.swift:17]: Spacing

File: Countries/Podfile [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
```
Comment (apolwork) [inline Countries/Podfile:2]: Do we need this comment? Or it should be removed?

File: Countries/Podfile [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+target 'Countries' do
+  # Comment the next line if you don't want to use dynamic frameworks
```
Comment (apolwork) [inline Countries/Podfile:5]: saa

File: README.md [hunk 6]
```diff
@@ -1 +1,12 @@
-# Countries
\ No newline at end of file
+# Countries
+## Test project by Seredovych Volodymyr 
+
+#### Building stages
+###### - install cocoapods
+###### - clone repository
+###### - navigate to folder with podfile
+###### - paste in console following line
```
Comment (apolwork) [inline README.md:8]: It is not necessary to mark list items as headers. Do you made this intentionally?

File: README.md [hunk 7]
```diff
@@ -1 +1,12 @@
...
+```sh
+pod install
+```
+###### - open Countries.xcworkspace to launch application
```
Comment (apolwork) [inline README.md:12]: saa

**Review by apolwork — CHANGES_REQUESTED** [review]

File: Countries/Countries/AppDelegate.swift (hunk 1 above)
Comment (Globalika) [reply Countries/Countries/AppDelegate.swift:78]: No. I will delete it in next PR

File: Countries/Countries/Queries.graphql (hunk 2 above)
Comment (Globalika) [reply Countries/Countries/Queries.graphql:5]: Yes. I will add new query in next PR

**Comment by Globalika** [comment]
fixed by apolwork comments*

File: Countries/.swiftlint.yml [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+ - vertical_parameter_alignment
+ - redundant_optional_initialization
+ - trailing_comma
+ - syntactic_sugar
+ - trailing_semicolon
+ - cyclomatic_complexity
+ - function_body_length
+ - comma
+
+excluded:
```
Comment (ambrusha) [inline Countries/.swiftlint.yml:30]: Add Countries/Countries/API.swift and Countries/R.generated.swift to excluded

File: Countries/Countries/AppDelegate.swift [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+//
+
+import UIKit
+import CoreData
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
```
Comment (ambrusha) [inline Countries/Countries/AppDelegate.swift:15]: one line not three...

Comment (ambrusha) [reply Countries/Countries/AppDelegate.swift:15]: And why `// MARK: - Core Data` still here?

File: Countries/Countries/Countries.xcdatamodeld/.xccurrentversion [hunk 10]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>_XCCurrentVersionName</key>
+	<string>Countries.xcdatamodel</string>
+</dict>
+</plist>
```
Comment (ambrusha) [inline Countries/Countries/Countries.xcdatamodeld/.xccurrentversion:8]: this file looks like for CoreData

File: Countries/Countries/Countries.xcdatamodeld/Countries.xcdatamodel/contents [hunk 11]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="1" systemVersion="11A491" minimumToolsVersion="Automatic" sourceLanguage="Swift" usedWithCloudKit="false" userDefinedModelVersionIdentifier="">
+    <elements/>
+</model>
```
Comment (ambrusha) [inline Countries/Countries/Countries.xcdatamodeld/Countries.xcdatamodel/contents:4]: this file looks like for CoreData

File: README.md [hunk 12]
```diff
@@ -1 +1,12 @@
-# Countries
\ No newline at end of file
+# Countries
+## Test project by Seredovych Volodymyr 
+
+#### Building stages
+- install cocoapods
+- clone repository
+- navigate to folder with podfile
+- paste in console following line
```
Comment (ambrusha) [inline README.md:8]:
```suggestion
- run from console `pod install`
```

File: Countries/Countries/AppDelegate.swift [hunk 13]
```diff
@@ -0,0 +1,14 @@
+//
+//  AppDelegate.swift
+//  Countries
+//
+//  Created by Volodymyr Seredovych on 23.11.2021.
+//
+
+import UIKit
+import CoreData
```
Comment (ambrusha) [inline Countries/Countries/AppDelegate.swift:9]: remove `import CoreData` please?

**Review by ambrusha — APPROVED** [review]

**Pull request merged**